specification/dimensions/servicename.md

```diff
@@ -1,10 +1,10 @@
 # Service Name
 
-Service Name is an identifier assigned to a service by the provider. The Service Name is commonly used for cost
-reporting, analysis and allocation scenarios.
+A service represents the offering that was purchased through the provider (e.g., professional or cloud services). For cloud resources, the service is a higher-level classification of the resource type and may include multiple usage charges of different types (e.g., compute, storage, networking). The service is not a classification of the type of usage. Instead, it represents the offering you might find on a provider's website.
 
-The ServiceName column MUST be present in the cost data. This column MUST be of type String and MUST NOT contain null
-values.
+The Service Name is a display name for the service or offering that was purchased. The Service Name is commonly used for scenarios like analyzing aggregate cost trends over time and filtering data to investigate anomalies.
```

Comment:
```suggestion
The Service Name is a display name for the offering that was consumed. The Service Name is commonly used for scenarios like analyzing aggregate cost trends over time and filtering data to investigate anomalies.
```

Reply:
In this context, "purchased" was referring to the acquisition of an offering, not the usage of it. We could change it to "acquire". We use purchase in the first sentence, so leaving that for now. Do people think we should change both?